src/Chainweb/CutDB.hs

```diff
@@ -407,15 +408,13 @@ startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
                     tableIterPrev it
                     go it
                 Left e -> throwM e
-                Right hm -> do
-                    hm' <- case _cutDbParamsInitialHeightLimit config of
+                Right hm -> unsafeMkCut v
+                    <$> case _cutDbParamsInitialHeightLimit config of
                         Nothing -> return hm
                         Just h -> limitCutHeaders wbhdb h hm
-                    logg Debug $ "joining intial cut from cut db configuration with cut that was loaded from the database " <> sshow hm'
-                    joinIntoHeavier_
-                        (_webBlockHeaderStoreCas headerStore)
-                        hm'
-                        (_cutMap $ _cutDbParamsInitialCut config)
+                            -- FIXME: this requires that the pact db is deleted because it
+                            -- interfers with rewind limit. As a fix temporarily disable
+                            -- rewind limit during initialization.
```

Comment:
This FIXME is unrelated to the change in this PR. But we discovered the issue while working on this PR. It's not urgent, since the initial cut limit is rarely used.